src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    margin-right: 20px;
+    display: flex;
+    align-items: center;
+  }
+
+  &__item:last-child {
+    margin: 0;
+  }
+
+  &__link {
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 60px;
+    color: $linkTextColor;
+    position: relative;
+  }
+
+  &__link:hover,
+  .is-active {
```

Comment:
The `.is-active` class should be nested inside the `&__link` block to follow the BEM methodology and to ensure that it's clear that `.is-active` is a modifier of the `&__link` element. Consider updating the selector to `&__link.is-active` to increase specificity and maintain consistency.